src/WebdriverClassicDriver.php

```diff
@@ -296,11 +296,11 @@ public function getText(
         #[Language('XPath')]
         string $xpath
     ): string {
-        return str_replace(
-            ["\r\n", "\r", "\n"],
+        return trim(str_replace(
+            ["\r\n", "\r", "\n", ' '],
```

Comment:
Is the `' '` a non-breaking space? I think I would rather use an escaped sequence (`"\xA0"`), otherwise it might not be obvious.

Reply:
@uuf6429 was writting the same, but if it's NBSP (non-breaking space) it should be U+202F

so it could
```php
mb_chr(8239)
``` 

or 

```php
chr(0xE2) . chr(0x80) . chr(0xAF)
```

no?

Reply:
Done. However, I had to use `"\xc2\xa0"`, because both `"\xA0"` and `chr(0xA0)` produced a broken UTF-8 symbol: `�`.

The https://stackoverflow.com/questions/40724543/how-to-replace-decoded-non-breakable-space-nbsp helped.

Reply:
Both editors I've had (PhpStorm and Sublime Text) were showing `0xa0` and not `0xc20xa0`:
![PhpStorm_NBSP](https://github.com/user-attachments/assets/7f83e9bb-2e54-422e-933e-4e4b2ba3c54b)
![SublimeText_NBSP](https://github.com/user-attachments/assets/f7bb0651-e1c4-4dee-b39e-e3a0926b2867)

In PHP you can't get it working without adding `\xc2` first. I wonder if it works in Selenium code at all without `\xC2`.

Reply:
I think I'm fine with it as long as the tests pass...should we maybe wait until the driver-testsuite one is merged and then rerun this?

Reply:
> I think I'm fine with it as long as the tests pass...should we maybe wait until the driver-testsuite one is merged and then rerun this?

Done.

See https://github.com/minkphp/webdriver-classic-driver/actions/runs/12132940669/job/33827642410?pr=52:

![GitHubActions_GetTextTest](https://github.com/user-attachments/assets/9df03784-ffaa-455b-b295-c5b428c65b2b)